amgut/db/patches/0010.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE ag.external_survey ( 
+	external_survey_id   bigserial  NOT NULL,
+	external_survey      varchar  NOT NULL,
+	external_survey_description varchar  ,
```

Comment:
sources is probably better, since it would also be good to store the URL here as well.